packages/kbn-apm-synthtrace/src/scenarios/degraded_logs.ts

```diff
@@ -30,10 +30,10 @@ const scenario: Scenario<LogDocument> = async (runOptions) => {
       const CLOUD_REGION = ['eu-central-1', 'us-east-1', 'area-51'];
 
       // "ignore_above": 1024 in mapping
-      const MALFORMED_LOG_LEVEL = MORE_THAN_1024_CHARS;
+      const DEGRADED_LOG_LEVEL = MORE_THAN_1024_CHARS;
 
       // "ignore_above": 1024 in mapping
-      const MALFORMED_CLOUD_REGION = MORE_THAN_1024_CHARS;
+      const DEGRADED_CLOUD_REGION = MORE_THAN_1024_CHARS;
```

Comment:
Can't you use `MORE_THAN_1024_CHARS` and not create these two extra variables?

Reply:
Addressed in [55814ec](https://github.com/elastic/kibana/pull/172586/commits/55814ec91335a449ed4a99829e76909d2f500dc0).